### PR TITLE
ListData and Font details features (GSoC 2023)

### PR DESCRIPTION
**Pull request by Codnelle**
This code contains ListData feature with some error in runtime.

### CODE REVIEW

**Comment by AppInventorWorkerBee** [comment]
Can one of the admins verify this patch?

**Comment by ewpatton** [comment]
@Codnelle Since this affects the companion we will need to update the base branch to `ucr` at some point (soon rather than later is better).

File: appinventor/components-ios/src/ListView.swift [hunk 1]
```diff
@@ -191,8 +250,17 @@ open class ListView: ViewComponent, AbstractMethodsForViewComponent,
   open func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
     let cell = tableView.dequeueReusableCell(withIdentifier: kDefaultTableCell) ??
       UITableViewCell(style: .default, reuseIdentifier: kDefaultTableCell)
+    
     cell.textLabel?.text = elements[indexPath.row]
     cell.textLabel?.font = cell.textLabel?.font.withSize(CGFloat(_textSize))
+    cell.textLabel?.font = cell.textLabel?.font.withSize(CGFloat(_fontSizeDetail))
```
Comment (ewpatton) [inline appinventor/components-ios/src/ListView.swift:256]: Note that the detail font size should be applied to cell.detailTextLabel

**Comment by ewpatton** [comment]
@AppInventorWorkerBee ok to test

**Pull request merged**